engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -315,6 +331,17 @@ public void setEnvironment(UniverseWrapper wrapper) {
         }
     }
 
+    public int findIndex(List<WorldSetupWrapper> worldsList,String worldName) {
```

Comment:
Needs javadoc (you can describe in the return tag what `-1` means for instance) and whitespace improvements.